pkgs/servers/monitoring/prometheus/default.nix

```diff
@@ -140,7 +126,6 @@ buildGoModule rec {
     description = "Service monitoring system and time series database";
     homepage = "https://prometheus.io";
     license = licenses.asl20;
-    maintainers = with maintainers; [ benley fpletz globin willibutz Frostman ];
-    platforms = platforms.unix;
+    maintainers = with maintainers; [ fpletz willibutz Frostman ];
```

Comment:
I looked at the commit history and looked who roughly did not commit something in ~1 year. Since the update is out for 6 weeks and no one started the rather trivial update, I thought I remove some inactive maintainers so that we are honest to ourselves.